### ISSUE TITLE
[Bug] Crash after loading an autosave of Zelda BOTW

### ISSUE THREAD

**Issue by bricevrs** [Ryujinx/Ryujinx#6726]
### Description of the issue

I'm trying to load the autosaved of Zelda BOTW but it crash just before showing the character. 

### Reproduction steps

Load an autosaved of Zelda

### Log file

Last login: Wed Apr 24 20:16:34 on ttys011
/Applications/Ryujinx\ 3.app/Contents/MacOS/Ryujinx ; exit;
brice@mbrice ~ % /Applications/Ryujinx\ 3.app/Contents/MacOS/Ryujinx ; exit;
00:00:00.003 |I| Configuration LogValueChange: EnableFileLog set to: True
00:00:00.045 |I| Configuration LogValueChange: ResScale set to: 1
00:00:00.050 |I| Configuration LogValueChange: ResScaleCustom set to: 2,6
00:00:00.050 |I| Configuration LogValueChange: MaxAnisotropy set to: -1
00:00:00.050 |I| Configuration LogValueChange: AspectRatio set to: Fixed16x10
00:00:00.051 |I| Configuration LogValueChange: BackendThreading set to: Auto
00:00:00.051 |I| Configuration LogValueChange: GraphicsBackend set to: Vulkan
00:00:00.052 |I| Configuration LogValueChange: PreferredGpu set to: 0x106B_0xE0303EF
00:00:00.052 |I| Configuration LogValueChange: AntiAliasing set to: SmaaMedium
00:00:00.053 |I| Configuration LogValueChange: ScalingFilter set to: Bilinear
00:00:00.053 |I| Configuration LogValueChange: ScalingFilterLevel set to: 100
00:00:00.053 |I| Configuration LogValueChange: EnableDockedMode set to: True
00:00:00.054 |I| Configuration LogValueChange: EnableVsync set to: True
00:00:00.054 |I| Configuration LogValueChange: EnableShaderCache set to: True
00:00:00.054 |I| Configuration LogValueChange: EnableTextureRecompression set to: True
00:00:00.054 |I| Configuration LogValueChange: EnableMacroHLE set to: True
00:00:00.054 |I| Configuration LogValueChange: EnableColorSpacePassthrough set to: False
00:00:00.054 |I| Configuration LogValueChange: EnablePtc set to: True
00:00:00.054 |I| Configuration LogValueChange: EnableInternetAccess set to: False
00:00:00.054 |I| Configuration LogValueChange: EnableFsIntegrityChecks set to: True
00:00:00.054 |I| Configuration LogValueChange: FsGlobalAccessLogMode set to: 0
00:00:00.054 |I| Configuration LogValueChange: AudioBackend set to: SDL2
00:00:00.054 |I| Configuration LogValueChange: AudioVolume set to: 1
00:00:00.055 |I| Configuration LogValueChange: MemoryManagerMode set to: HostMappedUnsafe
00:00:00.055 |I| Configuration LogValueChange: ExpandRam set to: False
00:00:00.055 |I| Configuration LogValueChange: IgnoreMissingServices set to: True
00:00:00.056 |I| Configuration LogValueChange: UseHypervisor set to: True
00:00:00.057 |I| Configuration LogValueChange: MultiplayerMode set to: Disabled
00:00:00.058 |N| Application PrintSystemInfo: Ryujinx Version: 1.1.1291
00:00:00.063 |N| Application Print: Operating System: macOS 14.3.1 (23D60) (Arm64)
00:00:00.063 |N| Application Print: CPU: Apple M2 Pro ; 12 logical
00:00:00.064 |N| Application Print: RAM: Total 16 GiB ; Available 6.9 GiB
00:00:00.066 |N| Application PrintSystemInfo: Logs Enabled: Info, Warning, Error, Guest, Stub
00:00:00.066 |N| Application PrintSystemInfo: Launch Mode: UserProfile
00:00:01.502 |E| .NET TP Worker 2024-04-26 23:48:53.098 Ryujinx[62764:5880682] WARNING: Secure coding is not enabled for restorable state! Enable secure coding by implementing NSApplicationDelegate.applicationSupportsSecureRestorableState: and returning YES.
00:00:00.099 |I| Gpu : Backend Threading (Auto): True
00:00:00.380 |N| Application LoadGuestApplication: Using Firmware Version: 17.0.0
00:00:00.381 |I| Application LoadGuestApplication: Loading as NSP.
00:00:00.395 |I| ModLoader QueryContentsDir: Searching mods for Application 01007EF00011E000 in "/Users/[redacted]/Library/Application Support/Ryujinx/mods/contents"
00:00:00.395 |I| ModLoader QueryContentsDir: Searching mods for Application 01007EF00011E000 in "/Users/[redacted]/Library/Application Support/Ryujinx/sdcard/atmosphere/contents"
00:00:00.403 |I| Loader Load: Loading rtld...
00:00:00.407 |I| Loader PrintRoSectionInfo: rtld:
    Module: nnrtld
00:00:00.407 |I| Loader Load: Loading main...
00:00:00.626 |I| Loader PrintRoSectionInfo: main:
    Module: D:\home\Cafe\U-King\trunk\Game\App\Rom\NX64\Product\code\U-King.nss
    SDK Libraries: SDK MW+Nintendo+NintendoWare_G3d-0_12_14-Release
                   SDK MW+Nintendo+NintendoWare_Vfx-0_12_13-Release
                   SDK MW+Nintendo+NintendoSDK_gfx-0_12_14-Release
                   SDK MW+Nintendo+NintendoWare_Font-0_12_14-Release
                   SDK MW+Nintendo+NintendoWare_Ui2d-0_12_14-Release
                   SDK MW+Nintendo+NintendoWare_Atk-0_12_14-Release
                   SDK MW+Havok+hcl_2014_1
                   SDK MW+Havok+hkp_2014_2
                   SDK MW+Havok+hkai_2014_2
                   SDK MW+Havok+hka_2014_2
00:00:00.626 |I| Loader Load: Loading subsdk0...
00:00:00.638 |I| Loader PrintRoSectionInfo: subsdk0:
    Module: nvn
    SDK Libraries: SDK MW+Nintendo+NintendoSDK_NVN-0_12_14-Release
00:00:00.638 |I| Loader Load: Loading subsdk1...
00:00:00.639 |I| Loader PrintRoSectionInfo: subsdk1:
    Module: nnDisplay
00:00:00.639 |I| Loader Load: Loading subsdk2...
00:00:00.640 |I| Loader PrintRoSectionInfo: subsdk2:
    Module: libcurl
    SDK Libraries: SDK MW+Nintendo+NintendoSDK_libcurl-0_12_14-Release
00:00:00.640 |I| Loader Load: Loading subsdk3...
00:00:00.670 |I| Loader PrintRoSectionInfo: subsdk3:
    Module: multimedia
    SDK Libraries: SDK MW+Nintendo+NintendoSDK_movie-0_12_14-Release
00:00:00.670 |I| Loader Load: Loading sdk...
00:00:00.704 |I| Loader PrintRoSectionInfo: sdk:
    Module: nnSdk
    FS SDK Version: 0.17.14
    SDK Libraries: SDK MW+Nintendo+NintendoSdk_nnSdk-0_12_14-Release
00:00:00.725 |I| Loader LoadNsos: Loading image 0 at 0x0000000008500000...
00:00:00.726 |I| Loader LoadNsos: Loading image 1 at 0x0000000008506000...
00:00:00.731 |I| Loader LoadNsos: Loading image 2 at 0x000000000a888000...
00:00:00.732 |I| Loader LoadNsos: Loading image 3 at 0x000000000aa7c000...
00:00:00.732 |I| Loader LoadNsos: Loading image 4 at 0x000000000aaa6000...
00:00:00.732 |I| Loader LoadNsos: Loading image 5 at 0x000000000aaeb000...
00:00:00.733 |I| Loader LoadNsos: Loading image 6 at 0x000000000b161000...
00:00:00.735 |I| ModLoader LoadCheats: Build ids found for application 01007EF00011E000:
    DB4B66DBF275A8261E2D51375F20C47D9A729537000000000000000000000000
    2094F152836683A3632C6153FD84AB8000000000000000000000000000000000
    9F7E69A5C5E8AA0E61199AC33DD0D2D13717C190000000000000000000000000
    FC3D9AAE8B6769E2D69961C005E239F536805F19000000000000000000000000
    2BA5FCC00BA28CDDA26C792E934949CBB5CBC513000000000000000000000000
    AA72F973A902C24747AF4522EE67EA9A7994C3FB000000000000000000000000
    99D55FB2A2CE9E14BD5D56428EC0397BE46E48EB000000000000000000000000
00:00:00.737 |I| Application EnsureSaveData: Ensuring required savedata exists.
00:00:00.769 |I| Loader Start: Application Loaded: The Legend of Zelda: Breath of the Wild v1.0.0 [01007ef00011e000] [64-bit]
00:00:00.788 |I| GUI.WindowThread Hid Configure: Configured Controller ProController to Player1
00:00:00.796 |I| GUI.WindowThread Hid SetupNpad: Connected Controller ProController to Player1
00:00:00.800 |E| .NET TP Worker [mvk-info] MoltenVK version 1.2.0, supporting Vulkan version 1.2.231.
00:00:00.801 |E| .NET TP Worker 	The following 83 Vulkan extensions are supported:
00:00:00.801 |E| .NET TP Worker 		VK_KHR_16bit_storage v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_8bit_storage v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_bind_memory2 v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_buffer_device_address v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_create_renderpass2 v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_dedicated_allocation v3
00:00:00.801 |E| .NET TP Worker 		VK_KHR_depth_stencil_resolve v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_descriptor_update_template v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_device_group v4
00:00:00.801 |E| .NET TP Worker 		VK_KHR_device_group_creation v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_driver_properties v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_dynamic_rendering v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_external_fence v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_external_fence_capabilities v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_external_memory v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_external_memory_capabilities v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_external_semaphore v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_external_semaphore_capabilities v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_fragment_shader_barycentric v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_get_memory_requirements2 v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_get_physical_device_properties2 v2
00:00:00.801 |E| .NET TP Worker 		VK_KHR_get_surface_capabilities2 v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_imageless_framebuffer v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_image_format_list v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_maintenance1 v2
00:00:00.801 |E| .NET TP Worker 		VK_KHR_maintenance2 v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_maintenance3 v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_multiview v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_portability_subset v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_push_descriptor v2
00:00:00.801 |E| .NET TP Worker 		VK_KHR_relaxed_block_layout v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_sampler_mirror_clamp_to_edge v3
00:00:00.801 |E| .NET TP Worker 		VK_KHR_sampler_ycbcr_conversion v14
00:00:00.801 |E| .NET TP Worker 		VK_KHR_separate_depth_stencil_layouts v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_shader_draw_parameters v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_shader_float_controls v4
00:00:00.801 |E| .NET TP Worker 		VK_KHR_shader_float16_int8 v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_shader_subgroup_extended_types v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_spirv_1_4 v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_storage_buffer_storage_class v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_surface v25
00:00:00.801 |E| .NET TP Worker 		VK_KHR_swapchain v70
00:00:00.801 |E| .NET TP Worker 		VK_KHR_swapchain_mutable_format v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_timeline_semaphore v2
00:00:00.801 |E| .NET TP Worker 		VK_KHR_uniform_buffer_standard_layout v1
00:00:00.801 |E| .NET TP Worker 		VK_KHR_variable_pointers v1
00:00:00.801 |E| .NET TP Worker 		VK_EXT_buffer_device_address v2
00:00:00.801 |E| .NET TP Worker 		VK_EXT_debug_marker v4
00:00:00.801 |E| .NET TP Worker 		VK_EXT_debug_report v10
00:00:00.801 |E| .NET TP Worker 		VK_EXT_debug_utils v2
00:00:00.801 |E| .NET TP Worker 		VK_EXT_descriptor_indexing v2
00:00:00.801 |E| .NET TP Worker 		VK_EXT_fragment_shader_interlock v1
00:00:00.801 |E| .NET TP Worker 		VK_EXT_hdr_metadata v2
00:00:00.801 |E| .NET TP Worker 		VK_EXT_host_query_reset v1
00:00:00.801 |E| .NET TP Worker 		VK_EXT_image_robustness v1
00:00:00.801 |E| .NET TP Worker 		VK_EXT_inline_uniform_block v1
00:00:00.801 |E| .NET TP Worker 		VK_EXT_memory_budget v1
00:00:00.801 |E| .NET TP Worker 		VK_EXT_metal_objects v1
00:00:00.801 |E| .NET TP Worker 		VK_EXT_metal_surface v1
00:00:00.801 |E| .NET TP Worker 		VK_EXT_post_depth_coverage v1
00:00:00.801 |E| .NET TP Worker 		VK_EXT_private_data v1
00:00:00.801 |E| .NET TP Worker 		VK_EXT_robustness2 v1
00:00:00.801 |E| .NET TP Worker 		VK_EXT_sample_locations v1
00:00:00.801 |E| .NET TP Worker 		VK_EXT_scalar_block_layout v1
00:00:00.801 |E| .NET TP Worker 		VK_EXT_separate_stencil_usage v1
00:00:00.801 |E| .NET TP Worker 		VK_EXT_shader_stencil_export v1
00:00:00.801 |E| .NET TP Worker 		VK_EXT_shader_viewport_index_layer v1
00:00:00.801 |E| .NET TP Worker 		VK_EXT_subgroup_size_control v2
00:00:00.801 |E| .NET TP Worker 		VK_EXT_swapchain_colorspace v4
00:00:00.801 |E| .NET TP Worker 		VK_EXT_texel_buffer_alignment v1
00:00:00.801 |E| .NET TP Worker 		VK_EXT_texture_compression_astc_hdr v1
00:00:00.801 |E| .NET TP Worker 		VK_EXT_vertex_attribute_divisor v3
00:00:00.801 |E| .NET TP Worker 		VK_AMD_gpu_shader_half_float v2
00:00:00.801 |E| .NET TP Worker 		VK_AMD_negative_viewport_height v1
00:00:00.801 |E| .NET TP Worker 		VK_AMD_shader_image_load_store_lod v1
00:00:00.801 |E| .NET TP Worker 		VK_AMD_shader_trinary_minmax v1
00:00:00.801 |E| .NET TP Worker 		VK_IMG_format_pvrtc v1
00:00:00.801 |E| .NET TP Worker 		VK_INTEL_shader_integer_functions2 v1
00:00:00.801 |E| .NET TP Worker 		VK_GOOGLE_display_timing v1
00:00:00.801 |E| .NET TP Worker 		VK_MVK_macos_surface v3
00:00:00.801 |E| .NET TP Worker 		VK_MVK_moltenvk v36
00:00:00.801 |E| .NET TP Worker 		VK_NV_fragment_shader_barycentric v1
00:00:00.801 |E| .NET TP Worker 		VK_NV_glsl_shader v1
00:00:00.801 |E| .NET TP Worker [mvk-info] GPU device:
00:00:00.801 |E| .NET TP Worker 		model: Apple M2 Pro
00:00:00.801 |E| .NET TP Worker 		type: Integrated
00:00:00.801 |E| .NET TP Worker 		vendorID: 0x106b
00:00:00.801 |E| .NET TP Worker 		deviceID: 0xe0303ef
00:00:00.801 |E| .NET TP Worker 		pipelineCacheUUID: 260BAD42-0E03-03EF-0000-000100000000
00:00:00.801 |E| .NET TP Worker 	supports the following Metal Versions, GPU's and Feature Sets:
00:00:00.801 |E| .NET TP Worker 		Metal Shading Language 2.4
00:00:00.801 |E| .NET TP Worker 		GPU Family Apple 7
00:00:00.801 |E| .NET TP Worker 		GPU Family Apple 6
00:00:00.801 |E| .NET TP Worker 		GPU Family Apple 5
00:00:00.801 |E| .NET TP Worker 		GPU Family Apple 4
00:00:00.801 |E| .NET TP Worker 		GPU Family Apple 3
00:00:00.801 |E| .NET TP Worker 		GPU Family Apple 2
00:00:00.801 |E| .NET TP Worker 		GPU Family Apple 1
00:00:00.801 |E| .NET TP Worker 		GPU Family Mac 2
00:00:00.801 |E| .NET TP Worker 		GPU Family Mac 1
00:00:00.801 |E| .NET TP Worker 		GPU Family Common 3
00:00:00.801 |E| .NET TP Worker 		GPU Family Common 2
00:00:00.801 |E| .NET TP Worker 		GPU Family Common 1
00:00:00.801 |E| .NET TP Worker 		macOS GPU Family 2 v1
00:00:00.801 |E| .NET TP Worker 		macOS GPU Family 1 v4
00:00:00.801 |E| .NET TP Worker 		macOS GPU Family 1 v3
00:00:00.801 |E| .NET TP Worker 		macOS GPU Family 1 v2
00:00:00.801 |E| .NET TP Worker 		macOS GPU Family 1 v1
00:00:00.801 |E| .NET TP Worker [mvk-info] Created VkInstance for Vulkan version 1.2.0, as requested by app, with the following 3 Vulkan extensions enabled:
00:00:00.801 |E| .NET TP Worker 		VK_KHR_surface v25
00:00:00.801 |E| .NET TP Worker 		VK_EXT_debug_utils v2
00:00:00.801 |E| .NET TP Worker 		VK_EXT_metal_surface v1
00:00:00.820 |E| .NET TP Worker [mvk-info] Vulkan semaphores using Metal implicit guarantees within a single queue.
00:00:00.820 |E| .NET TP Worker [mvk-info] Created VkDevice to run on GPU Apple M2 Pro with the following 8 Vulkan extensions enabled:
00:00:00.820 |E| .NET TP Worker 		VK_KHR_portability_subset v1
00:00:00.820 |E| .NET TP Worker 		VK_KHR_push_descriptor v2
00:00:00.820 |E| .NET TP Worker 		VK_KHR_shader_float16_int8 v1
00:00:00.820 |E| .NET TP Worker 		VK_KHR_swapchain v70
00:00:00.820 |E| .NET TP Worker 		VK_EXT_descriptor_indexing v2
00:00:00.820 |E| .NET TP Worker 		VK_EXT_fragment_shader_interlock v1
00:00:00.820 |E| .NET TP Worker 		VK_EXT_robustness2 v1
00:00:00.820 |E| .NET TP Worker 		VK_EXT_shader_stencil_export v1
00:00:00.872 |E| .NET TP Worker [mvk-info] Created 3 swapchain images with initial size (3022, 1914) and contents scale 2.0 for screen Built-in Retina Display.
00:00:00.873 |N| GUI.RenderThread Gpu PrintGpuInformation: Apple Apple M2 Pro (Vulkan v1.2.231, Driver v0.2.2008)
00:00:00.878 |I| GPU.MainThread Gpu LoadShaders: Loading 611 shaders from the cache...
00:00:05.992 |I| GPU.MainThread Gpu LoadShaders: Shader cache loaded.
00:00:06.047 |S| HLE.OsThread.51 ServiceAm GetAppletResourceUserId: Stubbed. {appletResourceUserId: 1}
00:00:06.050 |S| HLE.OsThread.51 ServiceAm AcquireForegroundRights: Stubbed. 
00:00:06.050 |S| HLE.OsThread.51 ServiceAm SetFocusHandlingMode: Stubbed. {unknownFlag1: False ; unknownFlag2: False ; unknownFlag3: True}
00:00:06.729 |G| HLE.OsThread.22 ServiceLm Log: Guest Log:
  Log level: Info
    Thread: MainThread
    Filename: D:/home/Cafe/U-King/trunk/Game/Lib/sead/engine/library/modules/src/basis/nin/seadRawPrintNin.cpp
    Line: 18
    Function: PrintStringImpl
    Message: 00000010

00:00:06.731 |S| HLE.OsThread.51 ServiceAm SetRestartMessageEnabled: Stubbed. {restartMessageEnabled: True}
00:00:06.732 |S| HLE.OsThread.51 ServiceAm SetOperationModeChangedNotification: Stubbed. {operationModeChangedNotification: True}
00:00:06.732 |S| HLE.OsThread.51 ServiceAm SetPerformanceModeChangedNotification: Stubbed. {performanceModeChangedNotification: True}
00:00:06.776 |I| HLE.OsThread.51 SurfaceFlinger CreateLayerFromId: Creating layer 1
00:00:06.789 |S| HLE.OsThread.8 ServiceNv InitializeEx: Stubbed. 
00:00:06.809 |S| HLE.OsThread.8 ServiceNv GetActiveSlotMask: Stubbed. 
00:00:06.840 |S| HLE.OsThread.8 ServiceNv SetNvMapFd: Stubbed. 
00:00:06.840 |S| HLE.OsThread.8 ServiceNv AllocGpfifoEx2: Stubbed. 
00:00:06.840 |S| HLE.OsThread.8 ServiceNv AllocObjCtx: Stubbed. 
00:00:06.841 |S| HLE.OsThread.8 ServiceNv SetErrorNotifier: Stubbed. 
00:00:06.841 |S| HLE.OsThread.8 ServiceNv SetUserData: Stubbed. 
00:00:06.841 |S| HLE.OsThread.8 ServiceNv SetPriority: Stubbed. 
00:00:06.842 |S| HLE.OsThread.8 ServiceNv ZcullBind: Stubbed. 
00:00:06.874 |S| HLE.OsThread.7 ServiceHid ActivateDebugPad: Stubbed. {appletResourceUserId: 1}
00:00:06.874 |S| HLE.OsThread.7 ServiceHid ActiveNpadImpl: Stubbed. {appletResourceUserId: 1 ; revision: 0}
00:00:06.875 |I| GUI.WindowThread Hid SetupNpad: Disconnected Controller ProController from Player1
00:00:06.876 |W| GUI.WindowThread Hid Remap: No matching controllers found. Application requests 'ProController, Handheld, JoyconPair, JoyconLeft, JoyconRight' on 'Player1, Player2, Player3, Player4, Player5, Player6, Player7, Player8, Handheld'
00:00:06.876 |S| HLE.OsThread.7 ServiceHid SetSupportedNpadIdType: Stubbed. 9 Players: Player1,Player2,Player3,Player4,Player5,Player6,Player7,Player8,Handheld
00:00:06.877 |S| HLE.OsThread.7 ServiceHid SetSupportedNpadStyleSet: Stubbed. {pid: 127 ; appletResourceUserId: 1 ; type: ProController, Handheld, JoyconPair, JoyconLeft, JoyconRight}
00:00:06.877 |I| GUI.WindowThread Hid SetupNpad: Connected Controller ProController to Player1
00:00:06.888 |S| HLE.OsThread.7 ServiceHid StartSixAxisSensor: Stubbed. {appletResourceUserId: 1 ; sixAxisSensorHandle: 131075}
00:00:06.910 |S| HLE.OsThread.51 ServiceAcc InitializeApplicationInfo: Stubbed. {TitleId: 72197163296350208}
00:00:09.500 |I| HLE.OsThread.15 AudioRenderer AcquireSessionId: Registered new renderer (0)
00:00:09.501 |I| HLE.OsThread.15 AudioRenderer Initialize: Initializing with REV1
00:00:09.510 |I| AudioProcessor.Worker AudioRenderer Work: Starting audio processor
00:00:09.510 |I| AudioRendererManager.Worker AudioRenderer SendCommands: Starting audio renderer
00:00:09.524 |I| HLE.OsThread.15 AudioRenderer Start: Starting renderer id 0
00:00:09.579 |I| AudioProcessor.Worker Audio EnsureAudioStreamSetup: New audio stream setup with a target sample count of 240
00:00:09.633 |S| HLE.OsThread.7 ServiceHid SetSupportedNpadStyleSet: Stubbed. {pid: 127 ; appletResourceUserId: 1 ; type: ProController, Handheld, JoyconPair}
00:00:10.111 |S| HLE.OsThread.7 ServiceHid SetGyroscopeZeroDriftMode: Stubbed. {appletResourceUserId: 1 ; sixAxisSensorHandle: 131075 ; _gyroscopeZeroDriftMode: Tight}
00:00:10.197 |E| .NET TP Worker [mvk-info] Created 3 swapchain images with initial size (3456, 2160) and contents scale 2.0 for screen Built-in Retina Display.
00:00:23.101 |I| HLE.OsThread.26 ServicePrepo ProcessPlayReport: 
PlayReport log:
 Kind: Normal
 Pid: 127
 ApplicationVersion: 0
 UserId: 00000000000000010000000000000000
 Room: option
 Report: {
    "BalloonTextOnOff": 1,
    "AutoSaveOnOff": 1,
    "CameraUpDownReverse": 0,
    "CameraRLReverse": 0,
    "TextRubyOnOff": 1,
    "JumpButtonChange": 0,
    "StickSensitivity": 2,
    "GyroOnOff": 1,
    "MiniMapDirection": 0,
    "AmiiboItemOnOff": 0,
    "MainScreenOnOff": 0,
    "TextWindowChange": 1,
    "DockInOnOff": 1,
    "ControllerType": 1,
    "AttachExtGrip": 0,
    "CtrlModeHandheldFree": 0,
    "CtrlModeHandheldExt": 0,
    "CtrlModeHandheldFullKey": 0,
    "CtrlModeHandheldHandheld": 0,
    "CtrlModeConsoleFree": 0,
    "CtrlModeConsoleExt": 0,
    "CtrlModeConsoleFull": 723,
    "PlayTimeHandheld": 0,
    "PlayTimeConsole": 723,
    "PlayTimeAll": 723
}

00:00:23.815 |W| HLE.OsThread.8 ServiceNv Wait: GPU processing thread is too slow, waiting on CPU...
00:00:30.212 |W| HLE.OsThread.8 ServiceNv Wait: GPU processing thread is too slow, waiting on CPU...
00:00:31.213 |E| HLE.OsThread.8 Gpu WaitOnSyncpoint: Wait on syncpoint 1 for threshold 1622 took more than 1000ms, resuming execution...
00:00:34.375 |W| HLE.OsThread.8 ServiceNv Wait: GPU processing thread is too slow, waiting on CPU...
00:00:36.149 |W| HLE.OsThread.8 ServiceNv Wait: GPU processing thread is too slow, waiting on CPU...
00:00:41.712 |I| HLE.GuestThread.58 Cpu PrintGuestStackTrace: Guest stack trace:
Process: The Legend of Zelda: Breath of the Wild, PID: 127
   0x0000000009792544	main:0x128c544	
   0x00000000095a24fc	main:0x109c4fc	
   0x000000000934431c	main:0xe3e31c	
   0x00000000096f1568	main:0x11eb568	
   0x00000000096f1974	main:0x11eb974	
   0x0000000008f24338	main:0xa1e338	
   0x0000000008f238cc	main:0xa1d8cc	
   0x000000000b1e93e0	sdk:0x883e0	
   0x000000000b1ec334	sdk:0x8b334	
   0x0000000000000000

00:00:41.713 |I| HLE.GuestThread.58 Cpu PrintGuestRegisterPrintout: Guest CPU registers:
	X[00]:	0x00000002289960c0
	X[01]:	0x0000000000000002
	X[02]:	0x00000002289960d8
	X[03]:	0x0000000228996020
	X[04]:	0x0000000000000009
	X[05]:	0x0000000205797030
	X[06]:	0x0000000000000000
	X[07]:	0x0000000000000000
	X[08]:	0x0000000000000001
	X[09]:	0x0000000000000000
	X[10]:	0x0000000000000010
	X[11]:	0x0000000228996040
	X[12]:	0x0000000000000000
	X[13]:	0x0000000000000000
	X[14]:	0x0000000000000000
	X[15]:	0x0000000205796e30
	X[16]:	0x000000000a173c10
	X[17]:	0x0000000000000000
	X[18]:	0x0000000000000000
	X[19]:	0x0000000000000009
	X[20]:	0x00000002057db100
	X[21]:	0x0000000000000000
	X[22]:	0x0000000205796df0
	X[23]:	0x00000002057bf880
	X[24]:	0x0000000228995cd0
	X[25]:	0x0000000000000000
	X[26]:	0x00000002289960d0
	X[27]:	0x000000000a855d68
	X[28]:	0x0000000000000009
	FP:	0x000000006d3cfc90 (SP:-0x01c0)
	LR:	0x0000000009798c38 (main:0x1292c38)	=> 
	SP:	0x000000006d3cfad0 (SP)
	PC:	0x0000000009799058 (main:0x1293058)	=> 

00:00:41.713 |E| HLE.GuestThread.58 Cpu InvalidAccessHandler: Invalid memory access at virtual address 0x0000000000000010.
00:00:41.745 |E| HLE.GuestThread.58 Application : Unhandled exception caught: Ryujinx.Memory.InvalidMemoryRegionException: Attempted to access an invalid memory region.
   at Ryujinx.Memory.Tracking.MemoryTracking.VirtualMemoryEvent(UInt64 address, UInt64 size, Boolean write, Boolean precise, Nullable`1 exemptId, Boolean guest) in /home/runner/work/Ryujinx/Ryujinx/src/Ryujinx.Memory/Tracking/MemoryTracking.cs:line 349
   at Ryujinx.Cpu.AppleHv.HvExecutionContext.DataAbort(MemoryTracking tracking, UInt64 vcpu, UInt32 esr) in /home/runner/work/Ryujinx/Ryujinx/src/Ryujinx.Cpu/AppleHv/HvExecutionContext.cs:line 240
   at Ryujinx.Cpu.AppleHv.HvExecutionContext.SynchronousException(HvMemoryManager memoryManager, HvVcpu& vcpu) in /home/runner/work/Ryujinx/Ryujinx/src/Ryujinx.Cpu/AppleHv/HvExecutionContext.cs:line 200
   at Ryujinx.Cpu.AppleHv.HvExecutionContext.Execute(HvMemoryManager memoryManager, UInt64 address) in /home/runner/work/Ryujinx/Ryujinx/src/Ryujinx.Cpu/AppleHv/HvExecutionContext.cs:line 159
   at Ryujinx.Cpu.AppleHv.HvCpuContext.Execute(IExecutionContext context, UInt64 address) in /home/runner/work/Ryujinx/Ryujinx/src/Ryujinx.Cpu/AppleHv/HvCpuContext.cs:line 27
   at Ryujinx.HLE.HOS.ArmProcessContext`1.Execute(IExecutionContext context, UInt64 codeAddress) in /home/runner/work/Ryujinx/Ryujinx/src/Ryujinx.HLE/HOS/ArmProcessContext.cs:line 62
   at Ryujinx.HLE.HOS.Kernel.Threading.KThread.ThreadStart() in /home/runner/work/Ryujinx/Ryujinx/src/Ryujinx.HLE/HOS/Kernel/Threading/KThread.cs:line 1264
   at System.Threading.Thread.StartCallback()
Unhandled exception. Ryujinx.Memory.InvalidMemoryRegionException: Attempted to access an invalid memory region.
   at Ryujinx.Memory.Tracking.MemoryTracking.VirtualMemoryEvent(UInt64 address, UInt64 size, Boolean write, Boolean precise, Nullable`1 exemptId, Boolean guest) in /home/runner/work/Ryujinx/Ryujinx/src/Ryujinx.Memory/Tracking/MemoryTracking.cs:line 349
   at Ryujinx.Cpu.AppleHv.HvExecutionContext.DataAbort(MemoryTracking tracking, UInt64 vcpu, UInt32 esr) in /home/runner/work/Ryujinx/Ryujinx/src/Ryujinx.Cpu/AppleHv/HvExecutionContext.cs:line 240
   at Ryujinx.Cpu.AppleHv.HvExecutionContext.SynchronousException(HvMemoryManager memoryManager, HvVcpu& vcpu) in /home/runner/work/Ryujinx/Ryujinx/src/Ryujinx.Cpu/AppleHv/HvExecutionContext.cs:line 200
   at Ryujinx.Cpu.AppleHv.HvExecutionContext.Execute(HvMemoryManager memoryManager, UInt64 address) in /home/runner/work/Ryujinx/Ryujinx/src/Ryujinx.Cpu/AppleHv/HvExecutionContext.cs:line 159
   at Ryujinx.Cpu.AppleHv.HvCpuContext.Execute(IExecutionContext context, UInt64 address) in /home/runner/work/Ryujinx/Ryujinx/src/Ryujinx.Cpu/AppleHv/HvCpuContext.cs:line 27
   at Ryujinx.HLE.HOS.ArmProcessContext`1.Execute(IExecutionContext context, UInt64 codeAddress) in /home/runner/work/Ryujinx/Ryujinx/src/Ryujinx.HLE/HOS/ArmProcessContext.cs:line 62
   at Ryujinx.HLE.HOS.Kernel.Threading.KThread.ThreadStart() in /home/runner/work/Ryujinx/Ryujinx/src/Ryujinx.HLE/HOS/Kernel/Threading/KThread.cs:line 1264
   at System.Threading.Thread.StartCallback()
zsh: abort      /Applications/Ryujinx\ 3.app/Contents/MacOS/Ryujinx

Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.
Deleting expired sessions...      11 completed.

[Process completed]

### OS

Mac os

### Ryujinx version

1.1.1291

### Game version

1.0.0

### CPU

M2 Pro

### RAM

16gb

**Comment by gdkchan** [comment]
Duplicate of #6727 (I know this one is slightly older, but the linked issue has more info so I'd prefer to keep that one open).
